app/assets/javascripts/components/features/notifications/components/notification.jsx

```diff
@@ -21,7 +21,7 @@ const Notification = React.createClass({
 
   renderFollow (account, link) {
     return (
-      <div className='notification'>
+      <div className='notification notification--follow'>
```

Comment:
Oh sorry, when I saw the class names with `__` I assumed you were following [BEM](http://getbem.com/introduction/). Will update.